apps/cowswap-frontend/src/modules/trade/state/priceImpactAtom.ts

```diff
@@ -8,3 +8,16 @@ export const priceImpactAtom = atom<PriceImpact>({
   // PriceImpactUpdater will set it to false anyway
   loading: true,
 })
+
+// Add a derived atom to prevent showing unknown price impact during initial loading
+export const stablePriceImpactAtom = atom<PriceImpact>((get) => {
```

Comment:
This atom returns the same in both cases

Reply:
Addressed in https://github.com/cowprotocol/cowswap/pull/5792/commits/f5b064452c5e77982c79c0ceae0a339277de3037

Eliminated all together.